File: easy/2_alphabet_triangle/alphabet_triangle.js
function alphabetTriangle() {
    for (let i = 'A'.charCodeAt(0); i <= 'G'.charCodeAt(0); i++) {
        for (let j = 'A'.charCodeAt(0); j <= i; j++) {
            process.stdout.write(String.fromCharCode(j) + ' ');
        }
        process.stdout.write('\n');
    }
}

alphabetTriangle();

module.exports = alphabetTriangle;

File: easy/2_alphabet_triangle/alphabet_triangle.go
package main

import "fmt"

func alphabetTriangle() {
	for i := 'A'; i <= 'G'; i++ {
		for j := 'A'; j <= i; j++ {
			fmt.Printf("%c ", j)
		}
		fmt.Println()
	}
}

func main() {
	alphabetTriangle()
}
